README.md

```diff
@@ -1,2 +1,18 @@
-# Hopp JS
-The official website for Hopp JS
\ No newline at end of file
+# hoppjs.com
+
+This is the repository for the official website for Hopp JS available at [hoppjs.com](http://hoppjs.com).
+
+# Set up for Contributing
```

Comment:
For contributing, just say "please make sure you read the contribution guide" (and link to it).